frontends/web/src/routes/account/send/send.tsx

```diff
@@ -819,7 +828,7 @@ class Send extends Component<Props, State> {
                       <small>{(proposedTotal && proposedTotal.unit) || 'N/A'}</small>
                     </span>
                     {(proposedTotal && proposedTotal.conversions) && (
-                      <span> <span className="text-gray">/</span> <strong>{proposedTotal.conversions[fiatUnit]}</strong> <small>{fiatUnit}</small></span>
+                      <span> <span className="text-gray">/</span> <strong>{proposedTotal.conversions[fiatUnit]}</strong> <small>{fiatUnit === 'BTC' ? btcUnit : fiatUnit}</small></span>
```

Comment:
Could save this in a variable and reuse it `const baseCurrencyUnit = fiatUnit === 'BTC' ? btcUnit : fiatUnit;` I am not sure about the name `baseCurrencyUnit`, but fiatUnit is a bit outdated as it can also be BTC.